Demo/Sources/Settings/SettingsView.swift

```diff
@@ -188,6 +189,26 @@ struct SettingsView: View {
         .frame(maxWidth: .infinity)
     }
 
+    @ViewBuilder
+    private func gitHubSection() -> some View {
+#if os(iOS)
+        Section("GitHub") {
+            Button("Project") { UIApplication.shared.open(Github.baseUrl().appending(path: "orgs/SRGSSR/projects/9")) }
```

Comment:
Would probably be nice to expose these endpoint URLs directly on `GitHub`. Same for the source code base URL. `GitHub.baseUrl()` can then be kept private.